hazelcast/src/main/java/com/hazelcast/kubernetes/KubernetesClient.java

```diff
@@ -617,8 +617,12 @@ private List<Endpoint> handleKnownException(RestClientException e) {
         } else if (e.getHttpErrorCode() == HTTP_FORBIDDEN) {
             if (!isKnownExceptionAlreadyLogged) {
                 LOGGER.warning("Kubernetes API access is forbidden! Starting standalone. To use Hazelcast Kubernetes discovery,"
-                        + " configure the required RBAC. For 'default' service account in 'default' namespace execute: "
-                        + "`kubectl apply -f https://raw.githubusercontent.com/hazelcast/hazelcast/master/kubernetes-rbac.yaml`");
+				 + " configure the required RBAC. For 'default' service account in 'default' namespace execute: "
+				 + "`kubectl apply -f https://raw.githubusercontent.com/hazelcast/hazelcast/master/kubernetes-rbac.yaml`" 
+				 + "If you want to use different service account and different namespace, you can update the mentioned rbac.yaml file accordingly and use it");
+				LOGGER.warning("Error Kubernetes API Cause: ", e.getCause());
+				LOGGER.warning("Error Kubernetes API Cause message: "+ e.getCause().getMessage());
```

Comment:
The formatting doesn't seem to be correct here. Moreover, it doesn't make sense to split the log into 3 messages. Could you use just the first one, but add the `e` as last parameter to the `warning(..., e);` method?